common/app/model/content.scala

```diff
@@ -83,8 +83,8 @@ final case class Content(
   lazy val isImmersive = fields.displayHint.contains("immersive") || isImmersiveGallery || tags.isTheMinuteArticle || isExplore || isPhotoEssay
   lazy val isPaidContent: Boolean = tags.tags.exists{ tag => tag.id == "tone/advertisement-features" }
   lazy val campaigns: List[Campaign] = targeting.CampaignAgent.getCampaignsForTags(tags.tags.map(_.id))
-  lazy val isRecipeArticle: Boolean = atoms.fold(false)(a => a.recipes.nonEmpty) && ArticleWithStructuredRecipe.isSwitchedOn
-
+  lazy val hasRecipeAtom: Boolean = atoms.fold(false)(a => a.recipes.nonEmpty)
+  lazy val showNewRecipeDesign: Boolean = ArticleWithStructuredRecipe.isSwitchedOn && hasRecipeAtom
```

Comment:
Do you still need this switch, given that the AB test is also a switch? You could just check `ABNewRecipeDesign.switch.isSwitchedOn` for equivalence?